README.md

```diff
@@ -46,13 +46,17 @@ After the CA certificate has been extended, there are three methods for distribu
 1. Manually deleting `ca.pem` on agents and letting them download that file as part of the next Puppet agent run. The agent will download that file only if it is absent, so it must be deleted to use this method.
 1. Using a Puppet file resource to manage `ca.pem`. _Note: This method is only possible if the CA certificate has not yet expired because Puppet communications depend upon a valid CA certificate._
 
-There are also two complementary tasks to check the expiration date of the CA certificate or any agent certificates.
+There are also complementary tasks to check the expiration date of the CA certificat, agent certificates and the CA CRL.
```

Comment:
Nitpicks: spelling of "certificate".  I'm a fan of Oxford commas, so I think there should be one after "agent certificates."